gobblin-compaction/src/main/java/gobblin/compaction/mapreduce/MRCompactor.java

```diff
@@ -141,6 +142,13 @@
   public static final String COMPACTION_DATASETS_FINDER = COMPACTION_PREFIX + "datasets.finder";
   public static final String DEFAULT_COMPACTION_DATASETS_FINDER = TimeBasedSubDirDatasetsFinder.class.getName();
 
+
+  // Rename source directories as a compaction complete indication
+  public static final String COMPACTION_RENAME_SOURCE_DIR = COMPACTION_PREFIX + "rename.source.dir";
+  public static final boolean DEFAULT_COMPACTION_RENAME_SOURCE_DIR = false;
```

Comment:
This name looks like a variable for a directory but it turns out to be a boolean